### PR TITLE
#266: Alias constructors for applicative builders for Try, Eval, IO

### PR DESCRIPTION
**Pull request by anstaendig**
Issue: #266

### CODE REVIEW

**Comment by anstaendig** [comment]
Sure, I just thought it was just fitting for `Eval` not the rest. Will do!

File: kategory-effects/src/main/kotlin/kategory/effects/data/IO.kt [hunk 1]
```diff
@@ -114,7 +68,180 @@ sealed class IO<out A> : IOKind<A> {
...
+                op1: () -> A,
+                op2: () -> B,
+                op3: () -> C,
+                op4: () -> D): IO<Tuple4<A, B, C, D>> = applicative().tupled(
+                invoke(op1),
+                invoke(op2),
+                invoke(op3),
+                invoke(op4)).ev()
+
+        fun <A, B, C, D, E> merge(
```
Comment (raulraja) [inline kategory-effects/src/main/kotlin/kategory/effects/data/IO.kt]: Regarding codestyle we should follow the same conventions as in https://github.com/kategory/kategory/blob/master/kategory-core/src/main/kotlin/kategory/typeclasses/Applicative.kt. Alternatively we can change the ones in Applicative to match these one. I'll leave up to you.

Comment (anstaendig) [reply kategory-effects/src/main/kotlin/kategory/effects/data/IO.kt]: I don't see one clear way of formatting it in`Applicative.kt`.

**Comment by pakoito** [comment]
Thank you @anstaendig for the PR! I hope it helped you understand Kategory and FP a bit better :D Do ping us on Slack if you want another ticket, we can ramp up the difficulty :P

**Pull request merged**